Public/Src/FrontEnd/UnitTests/Script.Interpretation/Test.BuildXL.FrontEnd.Script.Interpretation.dsc

```diff
@@ -25,6 +25,8 @@ namespace Script.Interpretation {
             importFrom("BuildXL.FrontEnd").Sdk.dll,
         ],
         //increase weight for frequent timeout pip
-        runTestArgs: { weight: 2 },
+        runTestArgs: { 
+            weight: 2,

```

Comment:
> weight: 2, [](start = 11, length = 11)

It's hard to say how much this actually helps but you might also try increasing this to something more dramatic like 8. Our test timeouts in our internal CloudBuild based runs have actually gotten quite a bit better recently but this assembly still had a few instances of bumping up to our timeout there in the last week